BlackLagerMessage.proto

```diff
@@ -1,5 +1,47 @@
 syntax = "proto3";
 
+/*
+ * https://grpc.io/docs/protoc-installation/
```

Comment:
This comment is unnecessary for this file. Links for reference should be put in the README rather than a specific .proto file